mep/books/admin.py

```diff
@@ -44,6 +46,7 @@ class ItemAdmin(admin.ModelAdmin):
         })
     )
     readonly_fields = ('mep_id', 'borrow_count')
+    actions = ('export_to_csv')
```

Comment:
This needs to be either `('export_to_csv', )` or `['export_to_csv']`. (A tuple with a single element needs a comma to register as a tuple).